python/tests/test_stdlib.py

```diff
@@ -19,8 +19,8 @@ def test_text_ops(self):
         #self.assertEqualMettaRunnerResults(metta.run('!(parse "$X")'),
         #                                  [[(V("X"))]])
 
-        self.assertEqualMettaRunnerResults(metta.run('!(parse "\\"A\\"")'),
-                                           [[(ValueAtom("A"))]])
+        self.assertEqualMettaRunnerResults(metta.run('!(parse "\\"A"\\")'),
+                                                   [[(ValueAtom("A"))]])
```

Comment:
Quote is located in incorrect place:
```suggestion
        self.assertEqualMettaRunnerResults(metta.run('!(parse "\\"A\\"")'),
                                                   [[(ValueAtom("A"))]])
```

Reply:
Done